docs/content/docs/deployment/azure-frs.md

```diff
@@ -1,5 +1,94 @@
 ---
 title: Connect to a Fluid Service in Azure with FrsClient
 menuPosition: 2
-status: placeholder
 ---
+
+# Introduction
+
+Fluid provides an implementation of its backend service, known as the Azure Fluid Relay Service (FRS), for clients to connect to. As part of the onboarding process, you will be provided a tenant ID, a tenant key, and orderer and storage URLs that will be used to connect your application to this service instance. The framework also provides a client-side package, `frs-client`, that will take in these configuration values to quickly allow you set up your collaborative application against the service instance. This package handles the logic of connecting your [Fluid Container](../build/containers.md) to the service while keeping the container object itself service-agnostic. You can use one instance of this client to manage multiple containers. 
+
+Let's take a look at how to go about using the `frs-client` in your app!
+
+# Connecting to the Service
+
+To connect to our FRS instance, we first need to instaniate our `FrsClient`. This takes in, as configuration parameters, the tenant ID, orderer, and storage URLs that were provided as part of the FRS onboarding process. It also requires a token provider to generate the JWT token that will be used to authorize the current user against the service. The `InsecureTokenProvider` should only be used for testing purposes as it exposes the tenant key secret in your client-side code bundle. This should be replaced with an implementation of `ITokenProvider` that fetches the token from your own backend service that is responsible for signing it with the tenant key. 
+
+```typescript
+const config: FrsConnectionConfig = {
+    tenantId: "myFrsTenantId",
+    // IMPORTANT: this token provider is suitable for testing ONLY. It is NOT secure.
```

Comment:
Let's delete this comment since it clutters the object and we are adding some clear callout warnings below around InsecureTokenProvider.

```suggestion
```